build/charts/yorkie-cluster/templates/istio/ratelimit/redis/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if .Values.ratelimit.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
```

Comment:
How about change this to `{{ .Values.ratelimit.name }}-redis` to let user set resource name freely?